keyboards/sapuseven/macropad12/readme.md

```diff
@@ -0,0 +1,28 @@
+# macropad
+
+![macropad12](https://i.imgur.com/StqDhH8.png)
```

Comment:
```suggestion
![macropad12](https://i.imgur.com/StqDhH8h.png)
```

This uses a smaller, but still high res version of the image that is more suitable for github markdown rendering.